From 2a52c06e235bd79f91851121f53f7c1808fde321 Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Mon, 7 Dec 2015 17:03:02 -0500
Subject: x86emu: Squash a warning

Apologies, should have caught this one when applying the previous x86emu
patch.

Signed-off-by: Adam Jackson <ajax@redhat.com>

diff --git a/hw/xfree86/x86emu/ops.c b/hw/xfree86/x86emu/ops.c
index 551dca7..210f8ce 100644
--- a/hw/xfree86/x86emu/ops.c
+++ b/hw/xfree86/x86emu/ops.c
@@ -12078,7 +12078,7 @@ x86emuOp_opcFF_word_RM(u8 X86EMU_UNUSED(op1))
             break;
         case 2:                /* call word ptr ... */
             if (M.x86.mode & SYSMODE_PREFIX_DATA) {
-                destreg = DECODE_RM_LONG_REGISTER(rl);
+                destreg = (u16 *)DECODE_RM_LONG_REGISTER(rl);
                 DECODE_PRINTF("\n");
                 TRACE_AND_STEP();
                 push_long(M.x86.R_EIP);
-- 
cgit v0.10.2

